docs/rules/idiomatic/use-in-operator.md

```diff
@@ -39,12 +39,8 @@ This linter rule provides the following configuration options:
 
 ```yaml
 rules: 
-  style:
+  idiomatic:
     use-in-operator:
       # one of "error", "warning", "ignore"
       level: error
 ```
-
-## Related Resources
```

Comment:
I don't think we should remove this. While it's a style guide, this is still a good reference.

Reply:
Added it back 👍